In [1]:
import pickle

In [3]:
loaded_model=pickle.load(open("Finalized_model_linear.sav",'rb'))
result=loaded_model.predict([[0]])

C:\Users\HP\Anaconda3\lib\site-packages\sklearn\base.py:451: UserWarning: X does not have valid feature names, but LinearRegression was fitted with feature names
  "X does not have valid feature names, but"


In [4]:
result

array([[26777.3913412]])

Here on deployment phase, we did again loaded model and predicted salary for '0' years of experience. The answer is 26777 which nothing but a bias value already found. 
Note: At any stage, bias value can be taken as intial/starting value in code whenever it is required